econml/ortho_forest.py

```diff
@@ -646,32 +771,37 @@ def parameter_estimator_func(Y, T, X,
         return parameter_estimator_func
 
     @staticmethod
-    def moment_and_mean_gradient_estimator_func(Y, T, X, W,
-                                                nuisance_estimates,
-                                                parameter_estimate):
-        """Calculate the moments and mean gradient at points given by (Y, T, X, W)."""
-        # Return moments and gradients
-        # Compute residuals
-        Y_res, T_res = ContinuousTreatmentOrthoForest._get_conforming_residuals(Y, T, nuisance_estimates)
-        # Compute moments
-        # Moments shape is (n, d_T)
-        moments = (Y_res - np.matmul(T_res, parameter_estimate)).reshape(-1, 1) * T_res
-        # Compute moment gradients
-        mean_gradient = - np.matmul(T_res.T, T_res) / T_res.shape[0]
-        return moments, mean_gradient
+    def moment_and_mean_gradient_estimator_func_gen(global_residualization=False):
```

Comment:
This would also go away if you get rid of `global_residualization`

Reply:
same reasoning as above

Reply:
Made changes.